modules/foreman_subnet.py

```diff
@@ -194,7 +194,7 @@
     'network': 'network',
     'cidr': 'cidr',
     'mask': 'mask',
-    'from_ip': 'from',
+    'from_ip': 'from_',
```

Comment:
What is going on here? Can you elabortate a little on the nailgun change?

Reply:
https://github.com/SatelliteQE/nailgun/pull/612 changed it from `from` to `from_`. https://github.com/SatelliteQE/nailgun/issues/606 contains the reason for the change.